src/GraphQL/Utilities/Visitors/ISchemaNodeVisitor.cs

```diff
@@ -39,22 +39,22 @@ public interface ISchemaNodeVisitor
         /// <summary>
         /// Visits field of registered within the schema output graph type.
         /// </summary>
-        void VisitFieldDefinition(FieldType field, ISchema schema);
+        void VisitFieldDefinition(FieldType field, IObjectGraphType type, ISchema schema);
```

Comment:
Ooops. I missed fields from interfaces. Should be 
` void VisitFieldDefinition(FieldType field, IComplexGraphType type, ISchema schema);` + change switch block inside ` public static void Run(this ISchemaNodeVisitor visitor, ISchema schema)`

or make two methods:
```
   void VisitFieldDefinition(FieldType field, IObjectGraphType type, ISchema schema);
   void VisitFieldDefinition(FieldType field, IInterfaceGraphType type, ISchema schema);
```

Reply:
Done; please review two added methods:

- `void VisitFieldDefinition(FieldType field, IInterfaceGraphType type, ISchema schema);`
- `void VisitFieldArgumentDefinition(QueryArgument argument, FieldType field, IInterfaceGraphType type, ISchema schema);`

Reply:
Thanks. Done.